src/components/SearchBar/SearchBar.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { Box, TextField, IconButton } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+export const SearchContainer = styled(Box)(({ theme }) => ({
+  position: 'absolute',
+  width: '324px',
+  height: '67px',
+  left: '929px',
+  top: '91px',
+  background: 'rgba(250, 250, 250, 0.75)', // Fondo con opacidad
+  backdropFilter: 'blur(10px)', // Añadir blur para efecto vidrio
+  borderRadius: '15px',
+  display: 'flex',
+  alignItems: 'center',
+  boxSizing: 'border-box',
+  border: '1px solid rgba(255, 255, 255, 0.2)', // Borde sutil
+
+  [theme.breakpoints.down('sm')]: {
+    width: '342px',
+    height: '67px',
+    left: '24px',
+    top: '231.87px',
+  },
+}));
+
+export const SearchIconButton = styled(IconButton)(({ theme }) => ({
+  position: 'absolute',
+  width: '19px',
+  height: '19px',
+  left: '31px', // 960 - 929 = 31px desde el contenedor
+  top: '24px', // 115 - 91 = 24px desde el contenedor
+  padding: 0,
+  minWidth: 0,
+  minHeight: 0,
+
+  '&.MuiIconButton-root': {
+    padding: 0,
+    margin: 0,
+    borderRadius: 0,
+  },
+
+  '& .MuiTouchRipple-root': {
+    display: 'none',
+  },
+
+  '&:hover': {
+    backgroundColor: 'transparent',
+  },
+
+  '& img': {
```

Comment:
Image icon sizes are hardcoded ('17.81px' etc.). Prefer theme.spacing or MUI icon sizing tokens to ensure visual consistency and scalability.

**Sugerencia:**
```typescript
Use a consistent numeric value (no fractional px), or better use theme.spacing-based sizes: width: theme.spacing(2.5), height: theme.spacing(2.5), or use MUI's SvgIcon + IconButton combination.
```